src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.iOS/Airship.cs

```diff
@@ -22,6 +22,18 @@ public class Airship : UADeepLinkDelegate, IAirship
 
         private void Init()
         {
+            NSNotificationCenter.DefaultCenter.AddObserver(aName: (NSString)"com.urbanairship.channel.channel_created", (NSNotification notification) =>
+            {
+                string channelID = notification.UserInfo["com.urbanairship.channel.identifier"].ToString() ;
```

Comment:
these ToString calls shouldn't be necessary. if it's a compiler error otherwise, I'd prefer that you solve it by casting, e.g.:

```
string channelID = (string) notification.UserInfo["com.urbanairship.channel.identifier"];
```